Assets/StreamingAssets/Images/Inventory/Raw Copper.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Sprites>
+	<Sprite name="Raw Iron"     x="0" y="0" w="7" h="7" pixelPerUnit="7" />

```

Comment:
Why is the sprite in Raw_Copper.xml called Raw Iron? Check when copying and pasting